src/main/webapp/matches.html

```diff
@@ -3,11 +3,81 @@
   <head>
     <meta charset="UTF-8">
     <title>Friends of Friends</title>
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <!-- Popper JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Latest compiled JavaScript -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Matches</h1>
+    <div class="container-fluid">
+      <nav class="navbar navbar-expand-sm justify-content-center navbar-background">
+        <ul class="navbar-nav">
+          <li class="nav-item">
+            <img src="images/logo.jpg" class="rounded-circle" alt="logo" height="80" width="100">
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="profile.html">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="feed.html">Feed</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link nav-text-style" href="matches.html">Matches</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="content">
+        <h1>Matches</h1>
+        <div class="row row-cols-5">
+          <div class="col card card-container m-5">
+            <img class="card-img-top" src="images/placeholder.jpg" alt="Card image">
+            <div class="card-body">
+              <h4 class="card-title">Match 1 Name</h4>
+              <p class="card-text">Insert match 1 bio here</p>
+              <a href="#" class="btn see-profile-btn">See Profile</a>
+            </div>
+          </div>
+          <div class="col card card-container-two m-5">
+            <img class="card-img-top" src="images/placeholder.jpg" alt="Card image">
+            <div class="card-body">
+              <h4 class="card-title">Match 1 Name</h4>
+              <p class="card-text">Insert match 1 bio here</p>
+              <a href="#" class="btn see-profile-btn-two">See Profile</a>
```

Comment:
nit: instead of having two classes to apply alternating styles (`see-profile-btn` and `see-profile-btn-two`), you can use even-odd rules: https://www.w3.org/Style/Examples/007/evenodd.en.html